content/en/docs/install-setup/installing/boot/clouds/openshift.md

```diff
@@ -0,0 +1,196 @@
+---
+title: OpenShift
+linktitle: OpenShift
+description: Using Boot On OpenShift
+date: 2020-05-06
+publishdate: 2020-05-06
+lastmod: 2020-05-06
+weight: 200
+---
+
```

Comment:
Maybe we could add an infobox (something like this here https://jenkins-x.io/docs/install-setup/create-cluster/gke/#cluster-provisioning) which puts into perspective what using Jenkins X on OpenShift means.

AFAIU, this will help you to run `jx boot` on an OpenShift cluster, but there won't be any persistent storage nor Vault. Not sure about things like DNS and TLS.

I would love to be able to manage the users expectations properly here.

Reply:
Good idea, added a warning on that, should show in the preview.